conferences/abstracts/UKRI-cloud-workshop-abstract.md

```diff
@@ -8,6 +8,6 @@ Sarah Gibson (*The Alan Turing Institute*) on behalf of **The Turing Way Collabo
 
 We aim to support the scientific method across all fields of research using cloud computing platforms to provide reproducible, reliable and reusable analyses. We plan to achieve this through capturing computing environments without need for installation, facilitating access to advanced computing resources and providing a "Safe Haven" for private repositories.
```

Comment:
Sorry to be a pain, but would you mind reformatting the doc to have each sentence on a new line? They'll still format correctly but it's easier to leave a review.

I don't think this is in our CONTRIBUTING guidelines, so if you could add that in there that would be amazing! Thank you 🎉